### PR TITLE
Feature/myhabits localization

### PR DESCRIPTION
**Pull request by HryshkoAnton**
Add localization to My Habits page

### CODE REVIEW

File: src/app/component/eco-news/eco-news.component.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class EcoNewsComponent implements OnInit {
   ngOnInit() {
     this.subscribeToLangChange();
     this.bindLang(this.localStorageService.getCurrentLanguage());
+    console.log(this.translate);
```
Comment (Gusvarov) [inline src/app/component/eco-news/eco-news.component.ts:22]: remove console.log

File: src/app/component/user/components/profile/profile.component.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export class ProfileComponent implements OnInit, OnDestroy {
   ngOnInit() {
     this.subscribeToLangChange();
     this.bindLang(this.localStorageService.getCurrentLanguage());
+    console.log(this.translate);
```
Comment (Gusvarov) [inline src/app/component/user/components/profile/profile.component.ts:20]: remove console.log

File: src/app/component/eco-news/eco-news.component.ts (hunk 1 above)
Comment (HryshkoAnton) [reply src/app/component/eco-news/eco-news.component.ts]: done

File: src/app/component/user/components/profile/profile.component.ts (hunk 2 above)
Comment (HryshkoAnton) [reply src/app/component/user/components/profile/profile.component.ts]: done

**Review by Gusvarov — APPROVED** [review]

**Review by nvolodymyr — APPROVED** [review]

**Review by nvolodymyr — APPROVED** [review]
maybe add little description:
[
image
](url)

**Review by yulkats — APPROVED** [review]

**Comment by mplesha** [comment]
conflicts

**Pull request merged**